tests/Load/utils/insertUsersUtils.js

```diff
@@ -0,0 +1,184 @@
+                body: JSON.stringify({
+                    email,
+                    password,
+                    initials,
+                }),
+                params: this.utils.getJsonHeader(),
+            };
+
+            yield [request, email, password];
+        }
+    }
+
+    prepareBatch(batchSize) {
+        const generator = this.requestGenerator(batchSize);
+        const batch = [];
+        const userPasswords = {};
+
+        for (let requestIndex = 0; requestIndex < batchSize; requestIndex++) {
+            const {value, done} = generator.next();
+            if (done) break;
+            const [request, email, password] = value;
+            batch.push(request);
+            userPasswords[email] = password;
+        }
+
+        return [batch, userPasswords];
+    }
+
+    insertUsers(numberOfUsers) {
+        const batchSize = this.config.batchSize;
+
+        const users = [];
+
+        for (let createdUsers = 0; createdUsers < numberOfUsers; createdUsers += batchSize) {
+            const [batch, userPasswords] = this.prepareBatch(batchSize);
+
+            const responses = http.batch(batch);
+
+            responses.forEach((response) => {
+                const user = JSON.parse(response.body);
+                user.password = userPasswords[user.email];
+                users.push(user);
+            });
+        }
+
+        return users;
+    }
+
+    countRequestForRampingRate(
+        startRps,
+        targetRps,
+        duration
+    ) {
+        const acceleration = (targetRps - startRps) / duration;
+
+        return Math.round((startRps * duration + acceleration * duration * duration / 2));
+    }
+
+    prepareUsers() {
+        let totalRequest = this.config.usersToInsert;
+
+        if (this.config.autoDetermineUsersToInsert) {
+            totalRequest = this.countTotalRequest();
+        }
+
+        return this.insertUsers(totalRequest);
+    }
+
+    countTotalRequest() {
+        let smokeRequests = 0;
+        let averageRequests = 0;
+        let stressRequests = 0;
+        let spikeRequests = 0;
+
+        if (this.utils.getCLIVariable('run_smoke') !== 'false') {
+            smokeRequests = this.countSmokeRequest();
+        }
+        if (this.utils.getCLIVariable('run_average') !== 'false') {
+            averageRequests = this.countAverageRequest();
+        }
+        if (this.utils.getCLIVariable('run_stress') !== 'false') {
+            stressRequests = this.countStressRequest();
+        }
+        if (this.utils.getCLIVariable('run_spike') !== 'false') {
+            spikeRequests = this.countSpikeRequest();
+        }
+
+        return Math.round(
+            (smokeRequests +
+                averageRequests +
+                stressRequests +
+                spikeRequests
+            ) * this.additionalUsersRatio
+        );
+    }
+
+    countSmokeRequest() {
+        return this.smokeConfig.rps * this.smokeConfig.duration
+    }
+
+    countAverageRequest() {
```

Comment:
this and countStressRequest looks like they same, need to make one function, you can refactor it like I showed above